src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj

```diff
@@ -7,26 +7,12 @@
     <MinVerTagPrefix>core-</MinVerTagPrefix>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
-    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcNetClientPkgVer)" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
-    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcNetClientPkgVer)" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Grpc" Version="$(GrpcPkgVer)" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
-    <PackageReference Include="Grpc" Version="$(GrpcPkgVer)" />
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
     <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPkgVer)" />
     <PackageReference Include="Grpc.Tools" Version="$(GrpcToolsPkgVer)" PrivateAssets="all" />
+    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcNetClientPkgVer)" />
```

Comment:
I think the primary concern here is that this is probably a breaking change for some users. Consider .NET Framework users for whom things are working just fine today, but with this change they now have to configure the HttpClient for things to work. Worst part is that they'll only find this out at runtime.

Reply:
One idea we'd discussed in the past is to introduce separate OTLP/gRPC and OTLP/HTTP exporter packages. Something like

* `OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc`: This package would _only_ depend on the new `Grpc.Net.Client` as you've done here. The benefit of it being a new package is that it won't break existing users.
* `OpenTelemetry.Exporter.OpenTelemetryProtocol.Http`: This could be done as a separate effort. Mostly just provides symmetry, but also reduces the number of transitive dependencies for users who don't want to use gRPC.